.github/workflows/upload-algolia-api.py

```diff
@@ -94,6 +96,19 @@ def main():
     print(f"Uploading {len(records)} records to {index.name=}")  # noqa:T201
     index.save_objects(records)
 
+    # Methods documented on backend-specific docs pages aren't scraped by Quarto
+    # since we construct them programmatically.
+    # There is a hook in docs/backends/_templates/api.qmd that calls
+    # `dump_methods_to_json_for_algolia` that serializes all the backend methods
+    # to a backend-specific json file in docs/backends/
+    # (Not Pandas and Impala because those backend pages don't use the template)
+    #
+    # Here, we load those records and upload them to the Algolia index
+    for record_json in glob.glob("docs/backends/*.json"):
+        with open(record_json) as f:
+            records = json.load(f)
+        index.save_objects(records)
```

Comment:
How expensive are these calls? Is it worth trying to make a single call?

Reply:
If there's any effort required, I'd say let's leave it until we notice a massive slowdown caused by this (which may never actually occur!)

Reply:
nah, pretty easy to smoosh them together